index.html

```diff
@@ -177,7 +177,47 @@ <h2>Multi-Post Stories</h2>
                 <button class="btn-1">See Project</button>
               </div>
             </div>
+          </div>
+        </div>
+          <div class="sub-grid-4">
+            <div class="sub-flex-2">
+              <div class="box">
+                <input type="text">
+              </div>
+              <div class="story-title">
+                <div>
+                  <h2>Multi-Post Stories</h2>
+                  <p>A daily selection of privately<br> personalized read; no accounts<br> or sign-ups required, has been
+                  the<br> industry's standard dummy text<br> ever since the 1500s, when an<br> unknown printer took a
+                  took a standard<br> dummy text</p>
+                </div>
+                <div>
+                  <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+                <div>
+                  <button class="btn-1">See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
       </div>
+            <div class="sub-grid-5">
+              <h2 class="text-align">CONTACT US</h2>
+              <p class="text-align">If you have an application you are<br> interested in developing, a feature that you<br> need
+                built or a project that needs coding.<br> I’d love to help with it</p>
+                <form class="text-align" action="/Ademola-Portfolio" method="post">
+                  <ul>
+                    <li><input type="text" id="text_input" name="text_input" placeholder="Text input"></li>
+                    <li><input type="text" id="email" name="email" placeholder="scdk@yahoo.com"></li>
+                  </ul>
+                  <textarea name="user_message" id="msg" cols="30" rows="10">Enter text here</textarea>
+                </form>
```

Comment:
- [ ] Please try to stick to Figma design as much as possible, currently your From is different from the form provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="305" alt="Screenshot 2022-03-30 at 12 22 38" src="https://user-images.githubusercontent.com/77038610/160785936-9c35ce14-a23a-4ba8-a82d-aa11f13ab1d4.png"> | <img width="281" alt="Screenshot 2022-03-30 at 12 23 17" src="https://user-images.githubusercontent.com/77038610/160786033-5be6a8ef-e978-4eaa-829a-e7223498f5a4.png">

More specifically: 

- [ ] Please add padding inside the inputs, and make the font style the same as provided in Figma
- [ ] Please set the font weight less in `<p> `tag
- [ ] Please add button and style it as provided in Figma
- [ ] Please center the form in the document, 
- [ ] Please add footer with a list of icons and email
- [ ] After you are done with changes, pelase make sure that your form looks identical with the one on Figma (colors, fonts, margins, paddings)